yt_dlp/extractor/youtube/pot/README.md

```diff
@@ -0,0 +1,277 @@
+# YoutubeIE PO Token Provider Framework
+
+As part of the YouTube extractor, we have a framework for providing PO Tokens programmatically. This can be used in plugins.
+
+Refer to the [PO Token Guide](https://github.com/yt-dlp/yt-dlp/wiki/PO-Token-Guide) for more information on PO Tokens.
+
+## Public APIs
+
+- `yt_dlp.extractor.youtube.pot.cache`
+- `yt_dlp.extractor.youtube.pot.provider`
+- `yt_dlp.extractor.youtube.pot.builtin.utils`
```

Comment:
Shouldn't this be `pot.utils`? Why `pot.builtins.utils`?

Reply:
I had originally put it in with builtins as they were utils specific to provider implementations. But we could also have a single top level utils module too